pages/linux/rusnapshot.md

```diff
@@ -0,0 +1,28 @@
+# rusnapshot
+
+> BTRFS snapshotting utility written in Rust.
+> More information: <https://github.com/Edu4rdSHL/rusnapshot>.
+
+- Create a snapshot using a config file:
+
+`sudo rusnapshot --config {{path/to/config.toml}} --cr`
+
+- List created snapshots:
+
+`sudo rusnapshot --config {{path/to/config.toml}} --list`
+
+- Delete a snapshot by ID or the name of the snapshot:
+
+`sudo rusnapshot --config {{path/to/config.toml}} --del --id {{snapshot_id}}`
+
+- Delete all `hourly` snapshots:
+
+`sudo rusnapshot --config {{path/to/config.toml}} --list --keep {{0}} --clean --kind {{hourly}}`
+
+- Create a read-write snapshot:
+
+`sudo rusnapshot --config {{path/to/config.toml}} --cr --rw`
+
+- Restore a snapshot:
+
+`sudo rusnapshot --config {{path/to/config.toml}} --id {{snapshot_id}} --restore`
```

Comment:
Since the `--config` is always present, how about we clear it a little and shorten all but the first occurence?

```suggestion
`sudo rusnapshot -c {{path/to/config.toml}} --list`

- Delete a snapshot by ID or the name of the snapshot:

`sudo rusnapshot -c {{path/to/config.toml}} --del --id {{snapshot_id}}`

- Delete all `hourly` snapshots:

`sudo rusnapshot -c {{path/to/config.toml}} --list --keep {{0}} --clean --kind {{hourly}}`

- Create a read-write snapshot:

`sudo rusnapshot -c {{path/to/config.toml}} --cr --rw`

- Restore a snapshot:

`sudo rusnapshot -c {{path/to/config.toml}} --id {{snapshot_id}} --restore`
```